dtos/intervalaction.go

```diff
@@ -20,6 +20,7 @@ type IntervalAction struct {
 	Content      string  `json:"content,omitempty"`
 	ContentType  string  `json:"contentType,omitempty"`
 	AdminState   string  `json:"adminState" validate:"oneof='LOCKED' 'UNLOCKED'"`
+	AuthMethod   string  `json:"authMethod" validate:"oneof='' 'none' 'jwt'"`
```

Comment:
Value should be be all caps as in line above for consistency. I assume the empty value will work, but haven't seen it used prior. 

```suggestion
	AuthMethod   string  `json:"authMethod" validate:"oneof='' 'NONE' 'JWT'"`
```

Reply:
Changed to upper case.